spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    display_name { Faker::DragonBall.character }
```

Comment:
I was thinking of publishing `Faker::Crypto.people` and  `Faker::Crypto.jargon`

Reply:
Someone already created a bitcoin address faker: https://github.com/stympy/faker/blob/master/doc/bitcoin.md

Reply:
I bet they'd accept a PR if you wanted to add it.